tbtadm/controller.cpp

```diff
@@ -746,12 +750,17 @@ void tbtadm::Controller::acl()
 
 void tbtadm::Controller::add(const fs::path& dir)
 {
-    if (m_sl == 2)
+    switch (m_sl)
     {
-        m_out << "Adding to ACL on SL2 must be done together with device "
-                 "approval\n";
-        return;
+        case 2:
+            m_out << "Adding to ACL on SL2 must be done together with device "
+                    "approval\n";
+            return;
+        case 0:
+            m_out << "Adding to ACL is not relevant in SL0\n";
+            return;
     }
```

Comment:
case 3? (and possibly more)
This need a better handling of "non SL 1" modes (see line 564)

Reply:
I think this would be handled in the other patch series in PR #45 , I include this change because my patches based on it.

Reply:
Oh, so this depends on the other one. OK, let's mention it next time so it'll be clear in first place.